src/screens/runPlan/ListMode/PlanElementList.tsx

```diff
@@ -60,13 +60,17 @@ export class PlanElementList extends React.PureComponent<Props, State> {
   }
 
   renderItem = ({ item, index }: { item: PlanElement; index: number }) => (
-    <PlanElementListItem
-      student={this.state.student}
-      itemParent={this.props.itemParent}
-      item={item}
-      index={index}
-      currentTaskIndex={this.completedPlanItemCounter()}
-    />
+    <>
```

Comment:
Although the code is perfectly fine I would resign from in-jsx if to the simpler `if` guard.

```
if(item.completed) { 
  return null;
}

return (
       <PlanElementListItem
          student={this.state.student}
          itemParent={this.props.itemParent}
          item={item}
          index={index}
          currentTaskIndex={this.completedPlanItemCounter()}
        />
);
```

Reply:
This way, you will get rid of:
- in-jsx if
- unnecessary fragment `<></>` syntax
- extra indentation